tools/mason/MasonPublish.chpl

```diff
@@ -71,6 +72,36 @@ proc masonPublish(ref args: list(string)) throws {
       args.append(potentialPath);
     }
 
+    if createReg {
+      var pathReg = '';
+      if args[args.size-1] == '-c' || args[args.size-1] == '--create-registry' then
+        pathReg = args[args.size-2];
+      else if args[args.size-2] == '-c' || args[args.size-2] == '--create-registry' then
+        pathReg = args[args.size-1];
+      try! {
+        if isDir(pathReg) && isDir(pathReg + '/Bricks') then
+          throw new owned MasonError('Registry already exists at %s'.format(pathReg));
+        else {
+          mkdir(pathReg);
+          mkdir(pathReg + '/Bricks');
+          mkdir(pathReg + '/DummyProject');
+          runCommand('touch ' + pathReg + '/0.1.0.toml');
```

Comment:
I think library calls are better when possible (better error-handling and error messages) - could we instead use the IO module to `open` the files? You could wrap this in a `proc touch` helper function for a clean finish:


```chpl
touch(pathReg + '/0.1.0.toml');
touch(pathReg + '/README.md');
```